release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -69,6 +69,10 @@ You can also specify multiple regions and zones on a cluster after installation.
 ==== Changes to the default `install-config.yaml` file
 After you run the installation program for {product-title} on vSphere, the default `install-config.yaml` file now includes `vcenters` and `failureDomains` fields, so that you can choose to specify multiple datacenters, region, and zone information for your cluster. You can leave these fields blank if you want to install an {product-title} cluster in a vSphere environment that consists of single datacenter running in a VMware vCenter.
 
+[id="ocp-4-13-installation-minimum-required-permissions-azure"]
+==== Minimum required permissions for Azure Public Cloud to install and delete the {product-title} cluster
+In {product-title} {product-version}, you can use the minimum required permissions for Azure Public Cloud to install and delete the {product-title} cluster. These permissions are available for installer-provisioned infrastructure and user-provisioned infrastructure.
```

Comment:
In {product-title} {product-version}, instead of using the built-in roles, you can now define your custom role(s) to include the minimum required permissions for Azure Public Cloud to install and delete the {product-title} cluster. These permissions are available for installer-provisioned infrastructure and user-provisioned infrastructure.

Reply:
Addressed